resources/lib/utils.py

```diff
@@ -419,3 +419,13 @@ def download_external_sub(language, codec, url):
         f.write(r.content)
 
     return file_path
+
+def get_bitrate(enum_value):
+
+	''' Get the video quality based on add-on settings.
+		Max bit rate supported by server: 2147483 (max signed 32bit integer)
+	'''
+	bitrate = [500, 1000, 1500, 2000, 2500, 3000, 4000, 5000, 6000,
+			   7000, 8000, 9000, 10000, 12000, 14000, 16000, 18000,
+			   20000, 25000, 30000, 35000, 40000, 100000, 1000000, 2147483]
+	return bitrate[enum_value or 24] * 1000
```

Comment:
Indentation on this function is super screwy, and it's mixing tabs and spaces to do the indenting which isn't good.  We only want to use spaces ideally.  All the code is one indent too far, and it looks like the second lines of the bitrate list are indented more than they should be.   

```
./resources/lib/utils.py:423:1: E302 expected 2 blank lines, found 1
./resources/lib/utils.py:425:1: W191 indentation contains tabs
./resources/lib/utils.py:425:1: E101 indentation contains mixed spaces and tabs
./resources/lib/utils.py:425:2: E117 over-indented
./resources/lib/utils.py:426:1: W191 indentation contains tabs
./resources/lib/utils.py:426:1: E101 indentation contains mixed spaces and tabs
./resources/lib/utils.py:427:1: W191 indentation contains tabs
./resources/lib/utils.py:427:1: E101 indentation contains mixed spaces and tabs
./resources/lib/utils.py:428:1: W191 indentation contains tabs
./resources/lib/utils.py:428:1: E101 indentation contains mixed spaces and tabs
./resources/lib/utils.py:429:1: W191 indentation contains tabs
./resources/lib/utils.py:429:1: E101 indentation contains mixed spaces and tabs
./resources/lib/utils.py:429:7: E128 continuation line under-indented for visual indent
./resources/lib/utils.py:430:1: W191 indentation contains tabs
./resources/lib/utils.py:430:1: E101 indentation contains mixed spaces and tabs
./resources/lib/utils.py:430:7: E128 continuation line under-indented for visual indent
./resources/lib/utils.py:431:1: W191 indentation contains tabs
./resources/lib/utils.py:431:1: E101 indentation contains mixed spaces and tabs
```